src/Discord.Net.Commands/Attributes/Preconditions/RequireOwnerAttribute.cs

```diff
@@ -10,17 +10,19 @@ namespace Discord.Commands
     [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = true, Inherited = true)]
     public class RequireOwnerAttribute : PreconditionAttribute
     {
+        public override string ErrorMessage { get; set; }
+
         public override async Task<PreconditionResult> CheckPermissionsAsync(ICommandContext context, CommandInfo command, IServiceProvider services)
         {
             switch (context.Client.TokenType)
             {
                 case TokenType.Bot:
                     var application = await context.Client.GetApplicationInfoAsync();
                     if (context.User.Id != application.Owner.Id)
-                        return PreconditionResult.FromError("Command can only be run by the owner of the bot");
+                        return PreconditionResult.FromError(ErrorMessage ?? "Command can only be run by the owner of the bot");
                     return PreconditionResult.FromSuccess();
                 default:
-                    return PreconditionResult.FromError($"{nameof(RequireOwnerAttribute)} is not supported by this {nameof(TokenType)}.");                    
+                    return PreconditionResult.FromError($"{nameof(RequireOwnerAttribute)} is not supported by this {nameof(TokenType)}.");
```

Comment:
Not sure if this one also needs its own custom message or not considering the other token types.

Reply:
I don't think we want to encourage creation of non-Bot applications.